airtime_mvc/application/models/Show.php

```diff
@@ -1273,7 +1272,6 @@ public static function getPrevCurrentNextOld($utcNow)
                             "starts"                  => $rows[$i-1]['starts'],
                             "ends"                    => $rows[$i-1]['ends'],
                             "record"                  => $rows[$i-1]['record'],
-                            "image_path"             => $rows[$i-1]['image_path'],
```

Comment:
Need to fix this still; once the upgrade is introduced (and, more importantly, tested) I'll go and grab the relevant code from upstream to reintroduce the showart/instance description